pick.js

```diff
@@ -45,8 +47,9 @@ function copy (from, to) {
         }
```

Comment:
this is batched in with the `isRegExp` conditional... keypaths + regular expressions seems like a very bad idea, so I don't think you need what you say you need...? but I've been wrong before